wlutil/config.py

```diff
@@ -640,21 +686,42 @@ def __init__(self, dirs=None, paths=None):
 
             try:
                 self._initializeFromBase(self.cfgs[cfgName])
-
             except KeyError as e:
-                log.warning("Skipping " + str(cfgName) + ":")
-                log.warning("\tMissing required option '" + e.args[0] + "'")
-                del self.cfgs[cfgName]
-                continue
+                raise WorkloadConfigError(cfgName,
+                                          extra=f"Missing required option '{e.args[0]}'")
             except Exception as e:
-                log.warning("Skipping " + str(cfgName) + ": Unable to parse config:")
-                log.warning("\t" + repr(e))
-                del self.cfgs[cfgName]
-                raise
-                continue
+                raise WorkloadConfigError(cfgName,
+                                          extra="Unable to parse config: " + repr(e))
 
             log.debug("Loaded " + str(cfgName))
 
+    def _loadRecursive(self, cfgName):
```

Comment:
Where is this used? Isn't this just a copy of the loading from above?

Reply:
good catch. I think it was leftover from some half-finished idea. Removed now.